t/api/03-auth.t

```diff
@@ -49,6 +49,18 @@ subtest 'authentication routes for plugins' => sub {
 
 client($t, apikey => undef, apisecret => undef);
 
+subtest 'key+secret from env variables' => sub {
+    my $t_new = Test::Mojo->new('OpenQA::WebAPI');
+    $t_new->delete_ok('/api/v1/assets/1')->status_is(403, 'clean user agent has no credentials');
+    local $ENV{OPENQA_API_KEY} = 'PERCIVALKEY02';
+    local $ENV{OPENQA_API_SECRET} = 'PERCIVALSECRET02';
+    $t_new->delete_ok('/api/v1/assets/1')->status_is(403)
+      ->json_is('/error' => 'no api key', 'key read from env (but missing administrator level)');
+    delete $ENV{OPENQA_API_KEY};
+    delete $ENV{OPENQA_API_SECRET};
+    client($t, apikey => undef, apisecret => undef);
```

Comment:
Could you explain what is the indention with the last lines.  they are not really needed. I run the test and it just fine without anyways.

Reply:
Improved with https://github.com/os-autoinst/openQA/pull/6865#issuecomment-3619923485